engines/python/setup/djl_python/fastertransformer.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+#
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file
+# except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
+# BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+import fastertransformer as ft
+from djl_python import Input, Output
+import logging
+from typing import Optional
+
+
+class FasterTransformerService(object):
+    def __init__(self) -> None:
+        self.initialized = False
+
+    def initialize(self, properties):
+        self.tensor_parallel_degree = int(
+            properties.get("tensor_parallel_degree", 1))
+        self.pipeline_parallel_degree = int(
+            properties.get("pipeline_parallel_degree", 1))
+        self.dtype = properties.get("dtype", "fp32")
+        self.model_id = properties.get("model_id")
```

Comment:
Please also add model dir solution here. User may still want to have the full model in a tarball (JumpStart)

Reply:
Yes, will add after some testing.